packages/webdriver/src/command.ts

```diff
@@ -101,7 +101,7 @@ export default function (
 
             this.emit('result', { method, endpoint, body, result })
 
-            if (command === 'deleteSession') {
+            if (command === 'deleteSession' && !process.env.KEEP_WDIO_LOGGER) {
```

Comment:
As I mentioned in my post, let's define a `WDIO_WORKER_CID` environment variable within `@wdio/local-runner` and check against its existence here.

Reply:
kk, I am not sure what else the usage of WDIO_WORKER_ID would be, but using it anyways